content/docs/0.8.x/continuous_delivery/multi_stage/index.md

```diff
@@ -35,16 +35,68 @@ spec:
     - name: "production"
 ```
 
-### Approval Strategy
+### Definition of Sequence in a Stage
+
+After defining the stages, sequences can be added to a stage. A sequence is an ordered list of tasks that are triggered sequentially. A `sequence` has the properties:
```

Comment:
"the stages" (plural) - remove "the"

Reply:
"the properties" - change to "consists of the following properties"

Reply:
thx, fixed.